jerry-debugger/jerry-client-ws.py

```diff
@@ -817,48 +846,59 @@ def release_function(debugger, data):
     logging.debug("Function {0x%x} byte-code released", byte_code_cp)
 
 
-def enable_breakpoint(debugger, breakpoint):
-    if breakpoint.active_index < 0:
-        debugger.next_breakpoint_index += 1
+def enable_breakpoint(debugger, breakpoint, pending):
```

Comment:
Do we really need the third argument? can't we just get the info out of the `breakpoint` argument?